### PR TITLE
Add CI to trigger benchmarks in PR

### PR DESCRIPTION
**Pull request by Kerollmops**
By using `@meilisearch run benchmarks` in a PR comment

### CODE REVIEW

**Comment by Kerollmops** [comment]
@meilisearch run benchmarks

File: .github/workflows/trigger-benchmarks-on-message.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: Run and display benchmarks in a PR
```
Comment (curquiza) [inline .github/workflows/trigger-benchmarks-on-message.yml:1]:
```suggestion
name: Benchmarks (PR)
```

**Review by curquiza — APPROVED** [review]
bors merge

**Comment by curquiza** [comment]
bors merge

**Pull request merged**